packages/common/test/pipes/number_pipe_spec.ts

```diff
@@ -125,8 +149,20 @@ import {beforeEach, describe, expect, it} from '@angular/core/testing/src/testin
           expect(pipe.transform(5.1234, 'USD', 'Custom name')).toEqual('Custom name5.12');
         });
 
+        it('should return null for NaN', () => {
+          expect(pipe.transform(Number.NaN)).toEqual(null);
+        });
+
+        it('should return null for null', () => {
+          expect(pipe.transform(null)).toEqual(null);
+        });
+
+        it('should return null for undefined', () => {
+          expect(pipe.transform(undefined)).toEqual(null);
+        });
+
         it('should not support other objects', () => {
-          expect(() => pipe.transform({}))
+          expect(() => pipe.transform(<any>{}))
```

Comment:
nit: we typically use `as` casts.

```suggestion
          expect(() => pipe.transform({} as any))
```

Reply:
oops! this was based on the `should not support other objects` from `case_conversion_pipes_spec.ts`, I did not know about the new convention.

I will fix the casts am I adding/modifying.
Should I also update the existing `<xyz>` casts in the files I an touching?

Reply:
I tried enabling the `no-angle-bracket-type-assertion` lint, but it looks like the codebase is still using them in several places.
I guess that kind of "global" refactor is better handled in another (non-BC) pull request.

In the meantime I will fix them in the code I am touching, so that PR will be a little bit simpler ;)

Reply:
Yes that sounds like the correct approach, thanks!

Reply:
Oh by the way, these could be improved even further in the future using `// @ts-expect-error` instead of the `any` cast.